doc/htmldoc/conf.py

```diff
@@ -378,14 +376,14 @@ def patch_documentation(patch_url):
     print("Preparing patch...")
     try:
         git_dir = str(source_dir / ".git")
-        git_hash = check_output(f"GIT_DIR={git_dir} git rev-parse HEAD", shell=True, encoding='utf8').strip()
+        git_hash = subprocess.check_output(f"GIT_DIR={git_dir} git rev-parse HEAD", shell=True, encoding='utf8').strip()
```

Comment:
Minor change to avoid a mess with badly named dirs.
```suggestion
        git_hash = subprocess.check_output(f"GIT_DIR='{git_dir}' git rev-parse HEAD", shell=True, encoding='utf8').strip()
```